app.py

```diff
@@ -236,6 +243,9 @@ def api_image_get(id):
     if not filename:
         return "", 404
 
+    if not owner == username or not public:
```

Comment:
There is a problem here, how come the owner see their private images? :eyes:

Reply:
For owner's private files:
Owner = yes &public = no
Wouldn't it work as expected? Or do you mean something else

Reply:
@Mitul16 I think it should be resolved now.